tdrs-backend/tdpservice/email/tasks.py

```diff
@@ -75,6 +75,14 @@ def get_ofa_admin_user_emails():
         groups__in=Group.objects.filter(name__in=('OFA Admin', 'OFA System Admin'))
     ).values_list('email', flat=True).distinct()
 
+def get_system_owner_email():
+    """Return the email of the System Owner."""
+    try:
+        user_email = User.objects.filter(groups__name='System Owner').values_list('email', flat=True).distinct()
```

Comment:
Note: intention is to create the 'System Owner' manually. We can add a small script to create that